test/go/iterative_verifier_test.go

```diff
@@ -393,12 +393,12 @@ func (t *ReverifyStoreTestSuite) SetupTest() {
 func (t *ReverifyStoreTestSuite) TestAddEntryIntoReverifyStoreWillDeduplicate() {
 	pk1 := uint64(100)
 	pk2 := uint64(101)
-	// different references to schema.Table shouldn't cause an issue.
```

Comment:
I don't really understand this comment. I used the same copy of TableSchema and all seem to work fine...